functions/lambda.py

```diff
@@ -1,15 +1,77 @@
-#Square number without lambda
-def sum(a ):
+text = """
+
+What is lambda in python? 
+
+It is anonymous function 
+It can take many number of arguments
+It can only have one expression
+It is written as a single line of execution
+It does not support type annotations
+It can be immediately invoked (IIFE)
+
+
+Lambda functions are frequently used with
+higher-order functions, which take one or more functions as arguments 
+or return one or more functions.
+filter(), sort(), sorted(), min(), and max()
```

Comment:
can this flipped to a print statement so that we can execute `python functions/lambda.py` and see the output :)

```
print("""
What is lambda in Python?

- It is an anonymous function.
- It can take any number of arguments.
- It can only have one expression.
- It is written as a single line of execution.
- It does not support type annotations.
- It can be immediately invoked (IIFE).

Lambda functions are frequently used with higher-order functions, which:
- Take one or more functions as arguments.
- Return one or more functions.

Some examples of these higher-order functions are filter(), sort(), sorted(), min(), and max().
""")

```

Reply:
resolve this in follow up PR :) merging to unblock